### DISCUSSION TITLE
PNP Refiners not showing up

### DISCUSSION THREAD

**Discussion by huetvictoria** [microsoft-search/pnp-modern-search#3731] [General]
I have mapped my site columns to RefinableString, I have added those RefinableStrings to the Properties in my Search Results Web Part, I have reindexed both the Site Collection and the Document Library and there is data within the Site Columns. I am adding the RefinableStrings to my Refiners web parts and they are just not showing up. Please help!

**Comment by kasperbolarsen** [comment]
Hi, first step should be to verify that the Refiner wp is connected to the Results wp and the other way round.
If that is the case, please review the data in the RefinableString using one of the debug tool here, https://microsoft-search.github.io/pnp-modern-search/using-query-tools-to-verify-issues/
